docs/indexing_samples.md

```diff
@@ -0,0 +1,51 @@
+# Adding sample indexing to the index runner
+
+## Reusable code
+
+* The event loop in `main.py`
```

Comment:
do you mean index_runner/main.py or do you mean index_runner/es_indexers/main.py?

Reply:
The former